Sources/FluentKit/Migration/Migration.swift

```diff
@@ -1,6 +1,19 @@
+/// `Migration` can assist with database migration. It can choose
+/// to create a new database schema for migration, or drop an
+/// existing schema. These actions are done in-place, returning no
+/// values.
 public protocol Migration {
     var name: String { get }
+    /// Prepares new database schema for the migration.
+    /// - Parameters:
+    ///     - database: `Database` to be created.
+    /// - returns: An asynchronous `Void`.
     func prepare(on database: Database) -> EventLoopFuture<Void>
+    
+    /// Drops the database.
```

Comment:
A migration revert just reverts the changes in a migration, it doesn't drop a database on its own